Devel

maressl opened the pull request:
How can i use "both_click" option on 86sw2 switch (2way wireless switch)

{"cmd":"report","model":"86sw2","sid":"158d0001712dab","short_id":27876,"data":"{\"dual_channel\":\"both_click\"}"}

Marko

T0RST3N committed:
-fix: heartbeat no longer report status again

T0RST3N committed:
-fix: check 'cmd' wrong, must define befor

T0RST3N committed:
-fix: check 'cmd' wrong, must define befor

T0RST3N committed:
-fix: rollback, no error

T0RST3N committed:
-fix: rollback, no error

T0RST3N committed:
-fix: rollback, no error

T0RST3N commented:
Hello Marko,

sorry i have just find your pull request. i didnt understand, are you asking? i didnt find any codechange.

if this is reported:
{"cmd":"report","model":"86sw2","sid":"158d0001712dab","short_id":27876,"data":"{"dual_channel":"both_click"}"}
i can make a change in the next release, please feel free and open an issue :-)

greetings 
torsten

T0RST3N committed:
some changes and fixes

- change: Perl modul math::round no longer necessary
- change: reading round is now rnd_tmp, rnd_hum, rnd_bat
- change: reading battery is now batterystate
- change: reading battery is now battery_level
- fix: if GW has no SID reading stop
- fix: if temperature is [1][0].xx or -[1][0].xx now the leading number
is display
- added: support switch 86sw2 dual_channel click
